be/src/runtime/integer_overflow_arithmetics.h

```diff
@@ -101,6 +101,11 @@ inline int clz128(unsigned __int128 v) {
 }
 
 inline bool int128_mul_overflow(int128_t a, int128_t b, int128_t* c) {
+    if (a == 0 || b == 0) {
```

Comment:
Why need this if check?

Reply:
https://github.com/StarRocks/starrocks/blob/ed57e48fcf1681ff570108c7700b0fedc194f6fc/be/src/runtime/integer_overflow_arithmetics.h#L122
There may be dividing by 0 without the check.